app/web/src/newhotness/explore_grid/ExploreGrid.vue

```diff
@@ -296,6 +296,8 @@ const virtualizerOptions = computed(() => ({
   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
   getScrollElement: () => scrollRef.value!,
   estimateSize: (i: number) => rowHeights.value[i] ?? 0,
+  // The item key is essential for reactivity and was found by @vbustamante and his valiant
+  // efforts. Without this, the virtualizer will not re-compute, even with new components.
```

Comment:
I'd like to better understand the issue this solves. Good work!

Reply:
This is actually @vbustamante's work from a week ago. I think it the rows are the same length, but everything is re-sorted that's passed into the props, you can sometimes end up in a situation where the rows aren't re-sorted too.